post/views.py

```
@@ -0,0 +1,89 @@
+
+
+class PostDetailView(APIView):
+    @swagger_auto_schema(
+        operation_id="게시글 상세 조회",
+        operation_description="게시글 1개의 상세 정보를 조회합니다.",
+        responses={200: PostSerializer},
+    )
+    def get(self, request, post_id):
+        try:
+            post = Post.objects.get(id=post_id)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)